### PR TITLE
skipping low enrollment notice for HC4CC groups

### PR DESCRIPTION
**Pull request by harshaellanki**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket:  [Redmine](https://redmine.priv.dchbx.org/issues/102280)

# A brief description of the changes

Current behavior:
HC4CC groups are receiving low enrollment notice

New behavior:

HC4CC groups should not receive low enrollment notice as they are not subject to the 2/3 minimum participation 
requirement.

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

### CODE REVIEW

**Review by utkarsh7989 — DISMISSED** [review]
LGTM

File: components/benefit_sponsors/spec/models/benefit_sponsors/model_events/low_enrollment_notice_for_employer_spec.rb [hunk 1]
```diff
@@ -53,6 +53,26 @@
...
+        params = {subject_gid: benefit_sponsorship.to_global_id, evidence_key: :osse_subsidy, evidence_value: 'true', effective_date: TimeKeeper.date_of_record }
+        result = ::Operations::Eligibilities::Osse::BuildEligibility.new.call(params)
+        eligibility = benefit_sponsorship.eligibilities.build(result.success.to_h)
+        eligibility.save!
+        year = model_instance.start_on.year
+        allow(EnrollRegistry).to receive(:feature?).with("aca_shop_osse_subsidy_#{year}").and_return(true)
+        allow(EnrollRegistry).to receive(:feature_enabled?).with("aca_shop_osse_subsidy_#{year}").and_return(true)
+      end
+
+      it "should trigger notice event" do
```
Comment (TonyHasIdeas) [inline components/benefit_sponsors/spec/models/benefit_sponsors/model_events/low_enrollment_notice_for_employer_spec.rb:71]: should  `expect(subject.notifier).not_to receive(:notify)` be below the process_application_events call?

Comment (TonyHasIdeas) [reply components/benefit_sponsors/spec/models/benefit_sponsors/model_events/low_enrollment_notice_for_employer_spec.rb:71]: Also should the statement be labeled "should NOT trigger notice event"?

Comment (TonyHasIdeas) [reply components/benefit_sponsors/spec/models/benefit_sponsors/model_events/low_enrollment_notice_for_employer_spec.rb]: Alright great, in that case LGTM.  I

**Review by TonyHasIdeas — APPROVED** [review]
LGTM, outside of label name

**Review by TonyHasIdeas — APPROVED** [review]

**Review by utkarsh7989 — APPROVED** [review]

**Pull request merged**